### ISSUE TITLE
Can I use the same preparams to generate multiple keys?

### ISSUE THREAD

**Issue by majecty** [binance-chain/tss-lib#145]
I want to generate multiple keys in the same group.
Regenerating pre-parameters every time requires too much time.

Is it safe to reuse the pre-parameters?

**Comment by yycen** [comment]
Do you mean child key derivation that keygen once and can later derive child keys?
This feature in not in master branch now, you can chek [this](https://github.com/binance-chain/tss-lib/pull/140) for usage.

**Comment by pajikap145** [comment]
Same question here
Is it safe to reuse preparams for multiple keygens? or should i have to generate new preparams for each keygen?
Im not in a position to use bip32 derivation right now

**Comment by yycen** [comment]
Yes, for N, h1, h2, you can use for another keygen, for the same set of parties.

**Comment by majecty** [comment]
@yycen Sorry for rate response.
Thanks for your answer! It really helps me.
